src/main/java/org/broadinstitute/consent/http/service/DatabaseDataAccessRequestAPI.java

```diff
@@ -423,12 +423,13 @@ public File createApprovedDARDocument() throws IOException {
         if (CollectionUtils.isNotEmpty(elections)) {
             for (Election election : elections) {
                 Document dar = describeDataAccessRequestById(election.getReferenceId());
-                String profileName = researcherPropertyDAO.findPropertyValueByPK(dar.getInteger(DarConstants.USER_ID), DarConstants.PROFILE_NAME);
-                String institution = researcherPropertyDAO.findPropertyValueByPK(dar.getInteger(DarConstants.USER_ID), DarConstants.INSTITUTION);
-                String consentName = consentDAO.findConsentNameFromDatasetID(dar.get(DarConstants.DATASET_ID, ArrayList.class).get(0).toString());
-                Integer consentElectionId = electionDAO.getElectionConsentIdByDARElectionId(election.getElectionId());
-                Election consentElection = electionDAO.findElectionById(consentElectionId);
-                dataAccessReportsParser.addApprovedDARLine(darWriter, election, dar, profileName, institution, consentName, consentElection.getTranslatedUseRestriction());
+                if(dar != null) {
```

Comment:
nit: spacing here and throughout: 
```suggestion
                if (dar != null) {
```